components/form/introduction.rst

```diff
@@ -202,9 +206,17 @@ to bootstrap or access Twig and add the :class:`Symfony\\Bridge\\Twig\\Extension
     )));
     $formEngine = new TwigRendererEngine(array($defaultFormTheme));
     $formEngine->setEnvironment($twig);
+
+    // create a Session object from the HttpFoundation component
+    $session = new Session();
+
+    $csrfGenerator = new UriSafeTokenGenerator();
+    $csrfStorage = new SessionTokenStorage($session);
```

Comment:
I wonder if it wasn't better to use the `NativeSessionTokenStorage` to not depend on the HttpFoundation component here.